### PR TITLE
fix issue#98 : adding column_table_limit feature

### PR DESCRIPTION
**Pull request by karanankit01**
by default the column_table_limit is same as column_limit.
User can set the value of column_table_limit in type int.
as discussed in [issue#98](https://github.com/Koihik/LuaFormatter/issues/98)

### CODE REVIEW

File: docs/Style-Config.md [hunk 1]
```diff
@@ -325,6 +325,49 @@ x = {
     k3 = v3, k4 = v4
 }
 ```
+### column_table_limit
+
+type: int, default: column_limit
+
+The column limit of one line for table. Default value is same as column_limit value.
+
+```lua
```
Comment (tammela) [inline docs/Style-Config.md:334]: Change the examples to something like:
```
column_table_limit: column_limit
[...]
column_table_limit: 20
[...]
```

File: docs/Style-Config.md [hunk 2]
```diff
@@ -325,6 +325,49 @@ x = {
     k3 = v3, k4 = v4
 }
 ```
+### column_table_limit
+
+type: int, default: column_limit
+
+The column limit of one line for table. Default value is same as column_limit value.
```
Comment (tammela) [inline docs/Style-Config.md:332]:
```suggestion
The column limit of each line of a table. Default value the same as column_limit.
```

**Comment by tammela** [comment]
@karanankit01 squash the commits into 2.
One that adds the feature and one that adds the documentation.

Fixes for commits not yet merged should not go to `master`.

File: docs/Style-Config.md (hunk 1 above)
Comment (karanankit01) [reply docs/Style-Config.md:334]: done

File: docs/Style-Config.md (hunk 2 above)
Comment (karanankit01) [reply docs/Style-Config.md]: done

File: README.md [hunk 3]
```diff
@@ -90,6 +90,7 @@ break_before_table_rb: true
 chop_down_table: false
 chop_down_kv_table: true
 table_sep: ","
+column_table_limit: column_limit
```
Comment (Koihik) [inline README.md:93]: The default value of `column_table_limit` is not a reference to `column_limit `.
Consider this config file:
```yaml
column_limit: 120
```
The value of `column_table_limit` is 80, not 120.

Comment (karanankit01) [reply README.md:93]: ok, I figured it out.
I fixed it by checking the config file and updating its value with `column_limit` .

File: src/Config.cpp [hunk 4]
```diff
@@ -59,11 +60,20 @@ void Config::readFromFile(const string& file) {
...
     // Keys are always strings
+    bool given = false;
+    string CTL = "column_table_limit";
+    string CL = "column_limit";
     for (auto kv : n) {
         auto key = kv.first.as<string>();
         if (node_[key]) {
             node_[key] = kv.second;
         }
+        if(key == CTL) {
```
Comment (tammela) [inline src/Config.cpp:71]: this code is not formatted correctly.
use `clang-format`.

Comment (karanankit01) [reply src/Config.cpp]: Done.
Thanks, I will mind it from now.

**Pull request merged**